en/services/gimbal_v2.md

```diff
@@ -0,0 +1,282 @@
+# Gimbal Protocol (v2)
+
+> **Note** This version is marked as work-in-progress.
+  This means that it is still subject to change.
+
+<span></span>
+> **Note** This version supersedes [Gimbal Protocol v1](../services/gimbal.md)
+
+## Introduction
+
+Gimbal protocol v2 was developed due to [various issues](../services/gimbal.md#known_issues) with the version 1.
+The main issues were the overall coordination of gimbal inputs, unclear specification and missing functionality.
+
+The issues have been addressed using the concept of gimbal manager and gimbal device, along with new messages to support these concepts and additional required functionality. 
+
+## Concepts
+
+### Gimbal Manager and Gimbal Device
+
+To accommodate for gimbals with varying degrees of capabilities and various hardware setups, we split up "the gimbal" into essentially two parts:
+- **The gimbal device**: the actual gimbal device, hardware and software.
+- **The gimbal manager**: software responsible to deconflict different gimbal messages and commands.
+
+This results in two message sets:
+- Gimbal manager message set: communication by anyone to gimbal manager.
+- Gimbal device message set: communication between gimbal manager and gimbal.
+
+Note that by default the gimbal manager is by default implemented by the autopilot.
+
```

Comment:
So I think the common setup diagrams are a bit ugly (due to mermaid). What is missing to make the whole thing easier to interpret is a high level sequence diagram here that shows 
1. The way it all behaves without saying where the gimbal manager lives
2. That uses the ROI message - because I think this is better for showing the "translating" going on. We could also mention the attitude messages, or have a separate example for that.

I think this will help when we get to common setups, because it will make it more clear that we're moving the manager around, but not changing the message flow.

What do you think - possible sequence below:

[![](https://mermaid.ink/img/eyJjb2RlIjoic2VxdWVuY2VEaWFncmFtO1xuICAgIHBhcnRpY2lwYW50IEdDU1xuICAgIHBhcnRpY2lwYW50IE1hbmFnZXJcbiAgICBwYXJ0aWNpcGFudCBHaW1iYWxcbiAgICBHaW1iYWwtPj5NYW5hZ2VyOiBHSU1CQUxfREVWSUNFX0lORk9STUFUSU9OXG4gICAgTWFuYWdlci0-PkdDUzogR0lNQkFMX01BTkFHRVJfSU5GT1JNQVRJT05cbiAgICBHQ1MtPj5NYW5hZ2VyOiBNQVZfQ01EX0RPX1NFVF9ST0lfTE9DQVRJT05cbiAgICBHQ1MtPj5NYW5hZ2VyOiBTdGFydCB0aW1lb3V0XG4gICAgTWFuYWdlci0-PkdDUzogQ09NTUFORF9BQ0tcbiAgICBNYW5hZ2VyLT4-R2ltYmFsOiBHSU1CQUxfREVWSUNFX1NFVF9BVFRJVFVERSAoc3RyZWFtKVxuICAgIEdpbWJhbC0-Pk1hbmFnZXI6IEdJTUJBTF9ERVZJQ0VfQVRUSVRVREVfU1RBVFVTIChzdHJlYW0pXG5NYW5hZ2VyLT4-R0NTOiBHSU1CQUxfREVWSUNFX0FUVElUVURFX1NUQVRVUyAoc3RyZWFtKVxuICAgICIsIm1lcm1haWQiOnsidGhlbWUiOiJkZWZhdWx0In0sInVwZGF0ZUVkaXRvciI6ZmFsc2V9)](https://mermaid-js.github.io/mermaid-live-editor/#/edit/eyJjb2RlIjoic2VxdWVuY2VEaWFncmFtO1xuICAgIHBhcnRpY2lwYW50IEdDU1xuICAgIHBhcnRpY2lwYW50IE1hbmFnZXJcbiAgICBwYXJ0aWNpcGFudCBHaW1iYWxcbiAgICBHaW1iYWwtPj5NYW5hZ2VyOiBHSU1CQUxfREVWSUNFX0lORk9STUFUSU9OXG4gICAgTWFuYWdlci0-PkdDUzogR0lNQkFMX01BTkFHRVJfSU5GT1JNQVRJT05cbiAgICBHQ1MtPj5NYW5hZ2VyOiBNQVZfQ01EX0RPX1NFVF9ST0lfTE9DQVRJT05cbiAgICBHQ1MtPj5NYW5hZ2VyOiBTdGFydCB0aW1lb3V0XG4gICAgTWFuYWdlci0-PkdDUzogQ09NTUFORF9BQ0tcbiAgICBNYW5hZ2VyLT4-R2ltYmFsOiBHSU1CQUxfREVWSUNFX1NFVF9BVFRJVFVERSAoc3RyZWFtKVxuICAgIEdpbWJhbC0-Pk1hbmFnZXI6IEdJTUJBTF9ERVZJQ0VfQVRUSVRVREVfU1RBVFVTIChzdHJlYW0pXG5NYW5hZ2VyLT4-R0NTOiBHSU1CQUxfREVWSUNFX0FUVElUVURFX1NUQVRVUyAoc3RyZWFtKVxuICAgICIsIm1lcm1haWQiOnsidGhlbWUiOiJkZWZhdWx0In0sInVwZGF0ZUVkaXRvciI6ZmFsc2V9)

Reply:
The sequence can be confusing because it's just one possibility :confused:...

Reply:
True, but you can explain that in text.